audit-ci.json

```diff
@@ -13,6 +13,7 @@
     "http-cache-semantics",
     "lodash.pick",
     "hoek",
+    "jsonpath-plus",
     "semver",
     "axios"
   ]
```

Comment:
So my commit removed many entries, and it's not reflected here.
https://github.com/nasa/cumulus/commit/a31481b9ae341b961217976f218aea649e7aa709#diff-3067268e04946afc99fcc6d204f22cc7105a282f106c6eda212a85d8dd3e5ba2L12

Reply:
@jennyhliu  good catch, I ... think.   Part of the confusion here is that there were commits in some of the backports (in order) that added them back.   I'm going to remove them and double check audit passes.     jsonpath-plus needs to be in the list though.

Reply:
81eb09b13f3